src/components/Sections/Library/LibraryPerson.js

```diff
@@ -9,76 +9,62 @@ import { withNamespaces } from 'react-i18next';
 import { formatError } from '../../../helpers/utils';
 import { actions, selectors } from '../../../redux/modules/assets';
 import { selectors as settings } from '../../../redux/modules/settings';
-import * as shapes from '../../shapes';
 import { ErrorSplash, FrownSplash, LoadingSplash } from '../../shared/Splash/Splash';
 
-class LibraryPerson extends Component {
-  static propTypes = {
-    sourceId: PropTypes.string.isRequired,
-    language: PropTypes.string.isRequired,
-    content: shapes.DataWipErr,
-    fetchAsset: PropTypes.func.isRequired,
-    t: PropTypes.func.isRequired,
-  };
+const getPerson = ({ fetchPerson, sourceId, language }) => {
+  fetchPerson(`persons-${sourceId}-${language}-html`);
```

Comment:
please build the slug inside the saga.
For example, lessons saga.

```
const language = yield select(state => settings.getLanguage(state.settings));
```

Reply:
I'm not sure that saga should decide what is language of my request. It can decide how to build request from parts that I supply it with. Language is my choice from site language, UI language and maybe something else (like Russian instead of Ukrainian)